scm/driver/github/user.go

```diff
@@ -31,8 +31,9 @@ func (s *userService) FindLogin(ctx context.Context, login string) (*scm.User, *
 }
 
 func (s *userService) FindEmail(ctx context.Context) (string, *scm.Response, error) {
-	user, res, err := s.Find(ctx)
-	return user.Email, res, err
```

Comment:
you can re-use the `ListEmail` function here:

```
func (s *userService) FindEmail(ctx context.Context) (string, *scm.Response, error) {
	out, res, err := s.ListEmails(ctx, scm.ListOptions{})
	return returnPrimaryEmail(out), res, err
}
```

Reply:
done

Reply:
done